internal/services/storage/storage_share_file_resource_test.go

```diff
@@ -325,14 +325,13 @@ func (r StorageShareFileResource) complete(data acceptance.TestData) string {
 %s
 
 resource "azurerm_storage_share_file" "test" {
-  name             = "test"
+  name             = "file"
   storage_share_id = azurerm_storage_share.test.id
 
 
   content_type        = "test_content_type"
   content_encoding    = "test_encoding"
   content_disposition = "test_content_disposition"
-  content_md5         = "1234567890abcdef1234567890abcdef"
```

Comment:
This attribute can't be updated and we use it in another test so it should be ok to remove here